ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java

```diff
@@ -57,9 +56,6 @@ public void initialize(final String appId, final Context context,
 
         mIsInitializing = true;
 
```

Comment:
if this function is not thread-safe ( and that a resaon of the bug) -  I believe we should fix the function too, by making mIsInitializing volatile or in any other way.

Reply:
Thanks, fixed.